op-batcher/batcher/channel_manager.go

```diff
@@ -111,6 +111,7 @@ func (s *channelManager) TxConfirmed(_id txID, inclusionBlock eth.BlockID) {
 	if channel, ok := s.txChannels[id]; ok {
 		delete(s.txChannels, id)
 		if timedOut := channel.TxConfirmed(id, inclusionBlock); timedOut {
+			s.log.Warn("channel timed out on chain", "channel_id", channel.ID(), "tx_id", id)
 			s.handleChannelInvalidated(channel)
```

Comment:
Definitely a missing log! Wondering if we also want to add a log to `handleChannelInvalidated` about which channels we drop, and what their statues were?

Reply:
I'll tackle this in a followup PR targeting [#batchermetric](https://github.com/ethereum-optimism/optimism/issues/14518)